docs/reference/commandline/service_create.md

```diff
@@ -136,6 +136,130 @@ $ docker service create \
 For more information about labels, refer to [apply custom
 metadata](../../userguide/labels-custom-metadata.md).
 
+### Volumes and bind-mounts (--mount)
+
+Services use a more advanced notation to 
+
+| Option                | Required          | Description
+|:----------------------|:------------------|:--------------------------------------------------------------------------------------------------------------------
+| **type**              |                   | The type of mount, can be either "bind", to bind-mount a directory or file from the host, or "volume" to use a managed volume. Defaults to "volume" if no type is specified.
```

Comment:
Here I would favor listing the available options:

`bind`:  bind-mounts a directory of file from a host into the container.
`volume`: default. mounts a managed volume (link to volume here?)